### PR TITLE
Tests for client

### PR DESCRIPTION
**Pull request by tarasiki**

### CODE REVIEW

File: EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs [hunk 1]
```diff
@@ -33,12 +33,28 @@ public MenuPage(IWebDriver driver) : base(driver)
...
+
+              
+        public void WaitForitemAddedPopUp(int timeToWait)
+        {
+            WaitElementIsEnable(timeToWait, _itemAddedPopUp);
+        }                    
+
+        public void ClearInputItemQuantity()
+        {
+            _inputItemQuantity.Clear(); ;
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs]: please remove useless ";" at the end

File: EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs [hunk 2]
```diff
@@ -41,7 +41,7 @@ public void WaitAndClickResturantList(int timeToWait)
         {
             _restaurantsList.WaitAndClick(driver, timeToWait);
         }
-
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs]: please remove empty spaces (пробіли)

File: EasyRestProjectNetTeam2/EasyRestTests/CheckRestaurantsInfoTests.cs [hunk 3]
```diff
@@ -59,7 +59,7 @@ public void CheckNavigateToBeerTagTest()
         {
             restaurantsPage = GetRestaurantsPage();
             restaurantsPage.WaitAndClickResturantList(dataModel.TimeToWait);
-            restaurantsPage.WaitAndClickBeerTag(dataModel.TimeToWait);
+            restaurantsPage.WaitAndClickBeerTag(dataModel.TimeToWait);
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckRestaurantsInfoTests.cs]: please remove empty spaces (пробіли) at the end

**Review by dmatsev — APPROVED** [review]
Approve with comments

**Review by SerhiiTarasiuk — APPROVED** [review]

**Review by Slim1011 — APPROVED** [review]

**Pull request merged**